commission/views/commission_settlement_views.xml

```diff
@@ -59,15 +59,16 @@
                 <header>
                     <button
                         string="Cancel"
-                        states="settled"
+                        invisible="state not in ['settled']"
```

Comment:
Simpler:

```suggestion
                        invisible="state != 'settled'"
```

Reply:
Dear @pedrobaeza

I applied the requested modification.

Concerning the file, It was copied to the directory by mistake and already removed from the repository.